x/evm/keeper/genesis.go

```diff
@@ -0,0 +1,21 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+)
+
+func (k Keeper) InitGenesis(ctx sdk.Context) {
```

Comment:
dont think it's necessary, but do we need to add migration logic?

Reply:
we will need migration logic to add this new module, but no custom state mutation logics are needed